crates/cli/src/commands/templates.rs

```diff
@@ -59,9 +66,17 @@ impl Options {
                 let captcha_config = CaptchaConfig::extract_or_default(figment)
                     .map_err(anyhow::Error::from_boxed)?;
 
-                let clock = SystemClock::default();
-                // XXX: we should disallow SeedableRng::from_entropy
-                let mut rng = rand_chacha::ChaChaRng::from_entropy();
+                let now = if stabilise {
+                    DateTime::from_timestamp_secs(0).unwrap()
```

Comment:
Could we use a more believable timestamp so that the IDs look like they are not all zeros?